Graphs/TreeGraph/index.js

```diff
@@ -474,6 +475,23 @@ class TreeGraph extends AbstractGraph {
 
         // ****************** Nodes section ***************************
 
+        const showToolTip = (d, showToolTip) => {
```

Comment:
these should not be inner functions! Move outside

Reply:
done